packages/frontend/editor-ui/src/views/Evaluations.ee/TestRunDetailView.vue

```diff
@@ -158,18 +156,13 @@ const fetchExecutionTestCases = async () => {
 };
 
 async function loadCachedUserPreferences() {
-	const cache = await indexedDbCache('workflows', 'evaluations');
```

Comment:
the change here is not exactly backwards compatible. But this would be cleaner to have one store for all workflows. Since it's a new feature that's not used a lot, we can afford to refactor this to use a new store instead of accruing tech debt of two stores.